primitive-types/Cargo.toml

```diff
@@ -13,8 +13,8 @@ uint = { version = "0.8.3", path = "../uint", default-features = false }
 impl-serde = { version = "0.3.0", path = "impls/serde", default-features = false, optional = true }
 impl-codec = { version = "0.4.1", path = "impls/codec", default-features = false, optional = true }
 impl-rlp = { version = "0.2", path = "impls/rlp", default-features = false, optional = true }
-# type-metadata = { version = "0.1", features = ["derive"], default-features = false, optional = true }
-type-metadata = { git = "https://github.com/type-metadata/type-metadata", default-features = false, features = ["derive"], optional = true }
+# scale-info = { version = "0.1", features = ["derive"], default-features = false, optional = true }
+scale-info = { git = "https://github.com/paritytech/scale-info", default-features = false, features = ["derive"], optional = true }
```

Comment:
Why don't you use the version `0.2` that can be found at [crates.io](crates.io/crates/scale-info)?

Reply:
(I'm back from vacation, sorry for the delay in reply)
this comment is on an outdated commit, this PR already uses 0.2

Reply:
Yep I just saw and by now there is version `0.3` on crates.io. :D